From 96576852a71ac51216b4c65684e7c19c5ad4963a Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Mon, 18 Apr 2016 18:48:17 +0800
Subject: [PATCH] ARM64: dts: rk3399: rename dts files

Rename the rk3399 dts files:
    rk3399-monkey.dts -> rk3399-evb1-android.dts
    rk3399-chrome.dts -> rk3399-evb1-cros.dts
    rk3399-tb.dtsi -> rk3399-evb.dtsi

Change-Id: Ie1f61d63b8fefc263a64d713d70947ceee8472c5
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/Makefile                       | 4 ++--
 .../boot/dts/rockchip/{rk3399-tb.dtsi => rk3399-evb.dtsi}   | 2 +-
 .../rockchip/{rk3399-monkey.dts => rk3399-evb1-android.dts} | 6 +++---
 .../rockchip/{rk3399-chrome.dts => rk3399-evb1-cros.dts}    | 6 +++---
 4 files changed, 9 insertions(+), 9 deletions(-)
 rename arch/arm64/boot/dts/rockchip/{rk3399-tb.dtsi => rk3399-evb.dtsi} (99%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-monkey.dts => rk3399-evb1-android.dts} (96%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-chrome.dts => rk3399-evb1-cros.dts} (93%)

diff --git a/arch/arm64/boot/dts/rockchip/Makefile b/arch/arm64/boot/dts/rockchip/Makefile
index 7acc0d85bf97..715c3c8bed85 100644
--- a/arch/arm64/boot/dts/rockchip/Makefile
+++ b/arch/arm64/boot/dts/rockchip/Makefile
@@ -2,11 +2,11 @@ dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3366-fpga.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3366-tb.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3368-r88.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3368-tb-sheep.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-chrome.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb1-android.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb1-cros.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-fpga.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-gru-gru.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-gru-kevin.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-monkey.dtb
 
 always		:= $(dtb-y)
 subdir-y	:= $(dts-dirs)
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-tb.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi
similarity index 99%
rename from arch/arm64/boot/dts/rockchip/rk3399-tb.dtsi
rename to arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi
index 8e278880abb4..99764306e28c 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-tb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi
@@ -44,7 +44,7 @@
 #include "rk3399.dtsi"
 
 / {
-	compatible = "rockchip,tb", "rockchip,rk3399";
+	compatible = "rockchip,evb", "rockchip,rk3399";
 
 	vdd_log: vdd-log {
 		compatible = "pwm-regulator";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-monkey.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb1-android.dts
similarity index 96%
rename from arch/arm64/boot/dts/rockchip/rk3399-monkey.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-evb1-android.dts
index c798a35ddb67..f7375fd195cc 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-monkey.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb1-android.dts
@@ -41,12 +41,12 @@
  */
 
 /dts-v1/;
-#include "rk3399-tb.dtsi"
+#include "rk3399-evb.dtsi"
 #include "rk3399-fb.dtsi"
 
 / {
-	model = "Rockchip monkey";
-	compatible = "rockchip,android", "rockchip,rk3399";
+	model = "Rockchip RK3399 Evaluation Board v1 (Android)";
+	compatible = "rockchip,android", "rockchip,rk3399-evb1", "rockchip,rk3399";
 
 	chosen {
 		bootargs = "console=uart,mmio32,0xff1a0000";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-chrome.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb1-cros.dts
similarity index 93%
rename from arch/arm64/boot/dts/rockchip/rk3399-chrome.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-evb1-cros.dts
index ca0c16f8e974..4947bef7ed7e 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-chrome.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb1-cros.dts
@@ -41,11 +41,11 @@
  */
 
 /dts-v1/;
-#include "rk3399-tb.dtsi"
+#include "rk3399-evb.dtsi"
 
 / {
-	model = "Rockchip Chrome";
-	compatible ="google,rk3399evb-rev2",  "google,rk3399evb-rev1", "google,rk3399evb-rev0", "rockchip,chrome", "rockchip,rk3399";
+	model = "Rockchip RK3399 Evaluation Board v1 (Chrome OS)";
+	compatible = "google,rk3399evb-rev1", "rockchip,chrome", "rockchip,evb1", "rockchip,rk3399";
 
 	/delete-node/ vdd-log;
 	/delete-node/ vdd-center;
-- 
2.35.3

